src/common/get-file-info.js

```diff
@@ -13,10 +19,9 @@ import inferParser from "../utils/infer-parser.js";
  * @returns {Promise<FileInfoResult>}
  *
  * Please note that prettier.getFileInfo() expects options.plugins to be an array of paths,
- * not an object. A transformation from this array to an object is automatically done
- * internally by the method wrapper. See withPlugins() in index.js.
+ * not an object.
  */
```

Comment:
The original comment is incorrect, `withPlugins` doesn't support object type of `plugins`.